### ISSUE TITLE
Purchase Receipt Emails: Invoice URL tag

### ISSUE THREAD

**Issue by flowdee** [quaderno/quaderno-edd#24]
It would be very helpful if the Quaderno invoice URL was available in the EDD Purchase Receipt emails as tag (e.g. `quaderno_invoice_url`) so we can link it directly in the receipts customers receive.

![Bildschirmfoto 2023-10-01 um 07 58 12](https://github.com/quaderno/quaderno-edd/assets/2278756/2391c835-4965-4cb7-96d5-02d8549f8dac)

Here's the code example from EDD how they easily add their stripe related tag:

```
/**
 * Register the statement_descriptor email tag.
 *
 * @since 2.6
 * @return void
 */
function edd_stripe_register_email_tags() {
	$statement_descriptor = edds_get_statement_descriptor();
	if ( ! empty( $statement_descriptor ) ) {
		edd_add_email_tag(
			'stripe_statement_descriptor',
			__( 'Outputs a line stating what charges will appear as on customer\'s credit card statements.', 'edds' ),
			'edd_stripe_statement_descriptor_template_tag',
			__( 'Statement Descriptor', 'edds' )
		);
	}
}
add_action( 'edd_add_email_tags', 'edd_stripe_register_email_tags' );

/**
 * Swap the {statement_descriptor} email tag with the string from the option
 *
 * @since 2.6
 * @param $payment_id
 *
 * @return mixed
 */
function edd_stripe_statement_descriptor_template_tag( $payment_id ) {
	$payment = new EDD_Payment( $payment_id );
	if ( 'stripe' !== $payment->gateway ) {
		return '';
	}

	$statement_descriptor = edds_get_statement_descriptor();
	if ( empty( $statement_descriptor ) ) {
		return '';
	}

	// If you want to filter this, use the %s to define where you want the actual statement descriptor to show in your message.
	$email_tag_output = __( apply_filters( 'edd_stripe_statement_descriptor_email_tag', 'Charges will appear on your card statement as %s' ), 'edds' );

	return sprintf( $email_tag_output, $statement_descriptor );
}
```

**Comment by flowdee** [comment]
@polimorfico 🙏🙂

**Comment by polimorfico** [comment]
Done! This new email tag is available as of version 1.35.0

**Comment by flowdee** [comment]
Thanks Carlos! @polimorfico
